### PR TITLE
Functions stuff

### PR DESCRIPTION
**Pull request by Meepedy**
I helped clean up some of your code and fix some bugs I found along the way. I commented on most of my commits in more detail about why I did what I did.

Some key notes:
- Created a main function that is called onload to ensure that everything loads before code is executed so their is no premature execution.
- You had questions 1-5 in functions but nothing was ever calling those functions so those questions were skipped over. I called them from within the main function mentioned above.
- Created functions for prompting the user's name and scoring at the end. Also moved the last two questions into functions of their own.
- Fixed it only accepting California as a correct answer for the last question, but sadly the while loop still doesn't break. I ran out of time to fix that.

### CODE REVIEW

**Comment by Meepedy** [comment]
obviously by no means do you have to merge any of my changes if you don't want to, just trying to help out.

File: app.js [hunk 1]
```diff
@@ -89,55 +89,58 @@ function question_five(){
...
-  }
-  else {
-    alert('I have not lived there, try again!');
-    i++;
+function question_seven(){
+  var lived = ['California', 'North Carolina', 'Mississippi', 'Florida' ];
+  var us_states = ['Alabama','Alaska','American Samoa','Arizona','Arkansas','California','Colorado','Connecticut','Delaware','District of Columbia','Federated States of Micronesia','Florida','Georgia','Guam','Hawaii','Idaho','Illinois','Indiana','Iowa','Kansas','Kentucky','Louisiana','Maine','Marshall Islands',
+    'Maryland','Massachusetts','Michigan','Minnesota','Mississippi','Missouri','Montana','Nebraska','Nevada','New Hampshire','New Jersey','New Mexico','New York','North Carolina','North Dakota','Northern Mariana Islands','Ohio','Oklahoma','Oregon','Palau','Pennsylvania','Puerto Rico','Rhode Island','South Carolina','South Dakota',
+    'Tennessee','Texas','Utah','Vermont','Virgin Island','Virginia','Washington','West Virginia','Wisconsin','Wyoming'];
+  attempts = 0;
```
Comment (dahlbyk) [inline app.js]: Good call - `attempts` is a better name here. I was going to remind you to use `var` here to keep `attempts` scoped to the current function, but I see that fixes comes in a later commit.

File: app.js [hunk 2]
```diff
@@ -125,19 +125,21 @@ function question_seven(){
...
     var states_guess = prompt('I have lived in four states other than Iowa, can you guess one?');
-    if (lived.indexOf(states_guess) === 0){
-      alert('That\'s correct! I\'ve lived in ' + lived + ' .Although I moved around a bit as a child and in the service, I consider Iowa my home!');
-      user_correct++;
-      break;
+    for (var i = 0; i < lived.length; i++){
+      if(lived[i] === states_guess){
+        alert('That\'s correct! I\'ve lived in ' + lived + ' .Although I moved around a bit as a child and in the service, I consider Iowa my home!');
+        user_correct++;
+        break;
```
Comment (dahlbyk) [inline app.js]: You might use `return` here to exit all the way out of the function once a correct answer is provided. As written, the code that follows will still run, so I'm pretty sure a correct answer on the last question will still result in the "Nice try" message.

File: app.js [hunk 3]
```diff
@@ -127,9 +127,9 @@ function question_seven(){
     'Tennessee','Texas','Utah','Vermont','Virgin Island','Virginia','Washington','West Virginia','Wisconsin','Wyoming'];
    var attempts = 0;
   while ( attempts < 7){
-    var states_guess = prompt('I have lived in four states other than Iowa, can you guess one?');
+    var states_guess = prompt('I have lived in four states other than Iowa, can you guess one?').toLowerCase();
```
Comment (dahlbyk) [inline app.js]: What if I cancel out of this `prompt`?

**Pull request merged**